docs/contracts/wsteth.md

```diff
@@ -3,25 +3,45 @@
 - [Source Code](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.6.12/WstETH.sol)
 - [Deployed Contract](https://etherscan.io/token/0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0)
 
-It's an ERC20 token that represents the account's share of the total
-supply of stETH tokens. WstETH token's balance only changes on transfers,
-unlike StETH that is also changed when oracles report staking rewards and
-penalties. It's a "power user" token for DeFi protocols which don't
-support rebasable tokens.
+## What is wrapped stETH (wstETH)?
 
-The contract is also a trustless wrapper that accepts stETH tokens and mints
-wstETH in return. Then the user unwraps, the contract burns user's wstETH
-and sends user locked stETH in return.
+It's an [ERC-20](https://eips.ethereum.org/EIPS/eip-20) value-accruing token
+wrapper for `stETH`. Its balance does not change with each oracle report, but its
+value in `stETH` does. Internally, it represents the user's share of the total
+supply of `stETH` tokens.
 
-The contract provides the staking shortcut: user can send ETH with regular
-transfer and get wstETH in return. The contract will send ETH to Lido submit
-method, staking it and wrapping the received stETH.
+## Why use wstETH?
+
+`wstETH` is mainly used as a layer of compatibility to integrate `stETH` into other
+DeFi protocols, that do not support rebasable tokens. Also, it may be used by
+users for tax optimization, because `stETH` [rebase](lido.md#rebase) may be
+a taxable event in some jurisdictions.
+
+## How to use wstETH?
+
+The contract can be used as a trustless wrapper that accepts stETH tokens and mints
```

Comment:
two points to consider:
- one can directly 'stake ether in wstETH' to bypass two-steps move
- wstETH is widely presented on L2s (can obtain the tokens there)